myproject/accounts/utils.py

```diff
@@ -2,6 +2,8 @@
 import os
 from .models import ExpiringToken
 
+# Contains a utility function to generate or refresh user tokens.
+
 def generate_token(user):
     token_key = binascii.hexlify(os.urandom(20)).decode()
```

Comment:
how to check the expiration time for the token 
when did you invalidate token 
you can use this 
https://django-rest-framework-simplejwt.readthedocs.io/en/latest/